README.md

```diff
@@ -61,7 +61,8 @@ Make sure you have registered your projects root under the version control prefe
 
 Open a project file that is under Git version control in the editor:
 
-      View → Open in Git host
+      View → Open in Git host or
+      Alt-F1 → Open in Git host
```

Comment:
I'm a little reluctant to include shortcuts in the README as they aren't always consistent between different OS.